modules/luci-base/luasrc/sys.lua

```diff
@@ -552,6 +553,39 @@ function wifi.getiwinfo(ifname)
 	return { ifname = ifname }
 end
 
+function wifi.checkkey()
+	ntm.init()
+	local dev
+	local rt = ""
+	local default_key = uci:get("system", "@system[0]", "default_key")
+
+	for _, dev in ipairs(ntm:get_wifidevs()) do
+		local net
```

Comment:
Setting this variable local is not needed. Actually the variable in the for loop is a completely different one even if they both are called "net". So if you'd set "local net = '123'" a print(net) after the for-loop would still output "123". Do you understand? loop-variables are always local-scope in Lua.

Reply:
Glass clear! 
I never used lua before and I got the wrong concept while looking at 
/usr/lib/lua/luci/tools/status.lua, so we may want to clear that code :)